File: examples/FaxSend.php
<?php

require_once __DIR__ . "/vendor/autoload.php";

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();

// Configure HTTP basic authorization: api_key
$config->setUsername("YOUR_API_KEY");

$faxApi = new Dropbox\Sign\Api\FaxApi($config);

$data = new Dropbox\Sign\Model\FaxSendRequest();
$data->setFiles([new SplFileObject(__DIR__ . "/example_signature_request.pdf")])
    ->setTestMode(true)
    ->setRecipient("16690000001")
    ->setSender("16690000000")
    ->setCoverPageTo("Jill Fax")
    ->setCoverPageMessage("I'm sending you a fax!")
    ->setCoverPageFrom("Faxer Faxerson")
    ->setTitle("This is what the fax is about!");

try {
    $result = $faxApi->faxSend($data);
    print_r($result);
} catch (Dropbox\Sign\ApiException $e) {
    $error = $e->getResponseObject();
    echo "Exception when calling Dropbox Sign API: "
        . print_r($error->getError());
}

File: examples/FaxSend.java
import com.dropbox.sign.ApiException;
import com.dropbox.sign.Configuration;
import com.dropbox.sign.api.*;
import com.dropbox.sign.auth.*;
import com.dropbox.sign.model.*;

import java.util.List;

public class Example {
    public static void main(String[] args) {
        var apiClient = Configuration.getDefaultApiClient()
            .setApiKey("YOUR_API_KEY");

        var faxApi = new FaxApi(apiClient);


        var data = new FaxSendRequest()
            .addFilesItem(new File("example_fax.pdf"))
            .testMode(true)
            .recipient("16690000001")
            .sender("16690000000")
            .coverPageTo("Jill Fax")
            .coverPageMessage("I'm sending you a fax!")
            .coverPageFrom("Faxer Faxerson")
            .title("This is what the fax is about!");

        try {
            FaxCreateResponse result = faxApi.faxSend(data);
            System.out.println(result);
        } catch (ApiException e) {
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}
